docs/technical_reference/unit_models/pressure_exchanger.rst

```diff
@@ -5,8 +5,8 @@ This pressure exchanger unit model:
     * is isothermal
     * supports a single liquid phase only
     * supports steady-state only
-    * assumes no mixing or leakage between the low and high pressure side
-    * assumes equal flowrates on both sides
+    * supports mixing or leakage between the low and high pressure side
+    * assumes equal flowrates on both side
```

Comment:
this should be changed to: enforces a constraint that the inlet flowrates to each side must be equal